sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -1531,6 +1537,37 @@ public void addContextTag(final @NotNull String contextTag) {
     this.contextTags.add(contextTag);
   }
 
+  /**
+   * Returns whether sending of client reports has been enabled.
+   *
+   * @return true if enabled; false if disabled
+   */
+  public @NotNull Boolean isSendClientReports() {
+    return sendClientReports;
+  }
+
+  /**
+   * Enables / disables sending of client reports.
+   *
+   * @param sendClientReports true enables client reports; false disables them
+   */
+  public void setSendClientReports(@NotNull Boolean sendClientReports) {
+    this.sendClientReports = sendClientReports;
+  }
+
+  /**
+   * Returns a ClientReportRecorder or a NoOp if sending of client reports has been disabled.
+   *
+   * @return a client report recorder or NoOp
+   */
+  public @NotNull ClientReportRecorder getClientReportRecorder() {
```

Comment:
Usually, We'd have a property in this class pointing to either one instance or another, See e.g. `logger`, Not really in favor of one or the other, but is there a reason to differ from the other implementations?

Reply:
Changed